algolia/internal/opt/attributes_to_transliterate_test.go

```diff
@@ -17,7 +17,7 @@ func TestAttributesToTransliterate(t *testing.T) {
 	}{
 		{
 			opts:     []interface{}{nil},
-			expected: opt.AttributesToTransliterate([]string{}...),
```

Comment:
diffs for `AttributesToTransliterate` is because of the same reason with the below PR.
https://github.com/algolia/algoliasearch-client-go/pull/657